src/frontend/src/pages/LandingPage/LandingPage.tsx

```diff
@@ -20,7 +20,11 @@ function LandingPage() {
     <div>
       <section className="hero-section wrapper">
         <div data-cy="landing" className="hero-content">
-          <img className="mobile-show-hero" src={hero} alt="Fill out PIA" />
+          <img
+            className="d-none d-md-block d-lg-none"
```

Comment:
why lg none? do we want to hide for larger screens as well?

Reply:
yes, the large screen will use the background picture

Reply:
ok cool. Thank you!

Reply:
shouldn't we only have d-lg-none? We want it to show for small screen.  This is public facing so it needs to render properly on mobile.